src/Features/Core/Portable/AddImport/CodeActions/AssemblyReferenceCodeAction.cs

```diff
@@ -17,22 +14,13 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
     {
             private class AssemblyReferenceCodeAction : AddImportCodeAction
             {
-                private readonly string _assemblyName;
-                private readonly string _fullyQualifiedTypeName;
-
                 private readonly Lazy<string> _lazyResolvedPath;
 
                 public AssemblyReferenceCodeAction(
                     Document originalDocument,
-                    ImmutableArray<TextChange> textChanges,
-                    string title,
-                    string assemblyName,
-                    string fullyQualifiedTypeName)
-                    : base(originalDocument, textChanges, title, WellKnownTagArrays.AddReference, CodeActionPriority.Low)
+                    AddImportFixData fixData)
```

Comment:
:bulb: If you don't use types derived from `AddImportFixData`, each of these code actions could benefit from a 'requires' like for the `fixData.Kind`.